soccer/gameplay/skills/move_direct.py

```diff
@@ -42,6 +43,12 @@ def threshold(self):
     def threshold(self, value):
         self._threshold = value
 
+    def velocity_check(self):
+        if (self.check_velocity):
+            return self.robot.vel.mag() < .05
```

Comment:
I feel like that might be a larger issue since there is a threshold to check if the robot's velocity is close to zero in like half of our plays